pkg/pxc/app/config/autotune.go

```diff
@@ -24,7 +30,14 @@ func getAutoTuneParams(q *res.Quantity) (string, error) {
 	// multiple of innodb_buffer_pool_chunk_size * innodb_buffer_pool_instances.
 	// More info: https://dev.mysql.com/doc/refman/8.0/en/innodb-buffer-pool-resize.html
 	if poolSize > int64(1000000000) {
```

Comment:
```suggestion
	if poolSize > int64(1073741824) {
```

According to my tests, `innodb_buffer_pool_instances` changes to 8 when `innodb_buffer_pool_size` is bigger than 1Gi, not 1G

Reply:
you're right

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1166/commits/1a176ec10e664aab61844bd637e036d18d7e46f9